website/README.md

```diff
@@ -0,0 +1,30 @@
+# fleetdm.com
+
+This is where the code for the public https://fleetdm.com website lives.
```

Comment:
For history going further back from this point, see https://github.com/fleetdm/fleetdm.com (the old original website repo prior to consolidating)